src/main/java/com/BrewMES/demo/model/BrewMES.java

```diff
@@ -56,7 +58,10 @@ public void connectMachine(String ipAddress) {
 			}
 			Machine newMachine = new Machine(ipAddress, connection);
 			machines.put(newMachine.getId(), newMachine);
-		} catch (Exception e) {
+		} catch (ExecutionException | InterruptedException e) {
+			e.printStackTrace();
+			Thread.currentThread().interrupt();
```

Comment:
Hvad er det for en Thread der skal stoppes? Oh hvorfor skal den ikke stoppes hvis der kommer en UaException?

Reply:
https://stackoverflow.com/questions/3976344/handling-interruptedexception-in-java

Reply:
Execution exception bør måske rykkes ned til UaException så Intterupted exception bliver catchet alene